plugins/modules/kafka_connectors.py

```diff
@@ -217,6 +224,7 @@ def run_module():
     module_args = dict(
         connect_url=dict(type='str', required=True),
         active_connectors=dict(type='list', elements='dict', required=True),
+        timeout=dict(type='int', required=False, default=60),
```

Comment:
Default value here and for the var are diff. Is that intended?

Reply:
I had set it to 60 initially, but after checking with customer made it 30 at other place, forgot to update it here. thanks for pointing out :)